releases/release-1.31/release-notes/maps/pr-124557-map.yaml

```
@@ -0,0 +1,5 @@
+pr: 124557
+releasenote:
+  text: |-
+    The nftables kube-proxy mode now has its own metrics rather than reporting
+    metrics with "iptables" in their names.
```

```suggestion
    Added metrics for the nftables kube-proxy mode rather than it reporting
    metrics with "iptables" in their names.
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653324198